x-pack/plugins/ml/public/application/jobs/new_job/pages/job_type/rare_job_icon.tsx

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+export const RareIcon = (
```

Comment:
this is currently a copy of the categorisation icon. a new icon needs designing and will be placed here when ready in a follow up PR